notebooks/sagemaker/Deploy rerank multilingual v3.0 model.ipynb

```diff
@@ -0,0 +1,294 @@
+    "    1. Either your IAM role has these three permissions and you have authority to make AWS Marketplace subscriptions in the AWS account used: \n",
+    "        1. **aws-marketplace:ViewSubscriptions**\n",
+    "        1. **aws-marketplace:Unsubscribe**\n",
+    "        1. **aws-marketplace:Subscribe**  \n",
+    "    2. or your AWS account has a subscription to [cohere-rerank-multilingual](https://aws.amazon.com/marketplace/pp/prodview-pf7d2umihcseq). If so, skip step: [Subscribe to the model package](#1.-Subscribe-to-the-model-package)\n",
+    "\n",
+    "## Contents:\n",
+    "1. [Subscribe to the model package](#1.-Subscribe-to-the-model-package)\n",
+    "2. [Create an endpoint and perform real-time inference](#2.-Create-an-endpoint-and-perform-real-time-inference)\n",
+    "   1. [Create an endpoint](#A.-Create-an-endpoint)\n",
+    "   2. [Create input payload](#B.-Create-input-payload)\n",
+    "   3. [Perform real-time inference](#C.-Perform-real-time-inference)\n",
+    "   4. [Delete the endpoint](#H.-Delete-the-endpoint)\n",
+    "3. [Clean-up](#4.-Clean-up)\n",
+    "    1. [Delete the model](#A.-Delete-the-model)\n",
+    "    2. [Unsubscribe to the listing (optional)](#B.-Unsubscribe-to-the-listing-(optional))\n",
+    "    \n",
+    "\n",
+    "## Usage instructions\n",
+    "You can run this notebook one cell at a time (By using Shift+Enter for running a cell)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Subscribe to the model package"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To subscribe to the model package:\n",
+    "1. Open the model package listing page [cohere-rerank-multilingual](https://aws.amazon.com/marketplace/pp/prodview-pf7d2umihcseq)\n",
+    "1. On the AWS Marketplace listing, click on the **Continue to subscribe** button.\n",
+    "1. On the **Subscribe to this software** page, review and click on **\"Accept Offer\"** if you and your organization agrees with EULA, pricing, and support terms. \n",
+    "1. Once you click on **Continue to configuration button** and then choose a **region**, you will see a **Product Arn** displayed. This is the model package ARN that you need to specify while creating a deployable model using Boto3. Copy the ARN corresponding to your region and specify the same in the following cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install --upgrade cohere-aws\n",
+    "# if you upgrade the package, you need to restart the kernel\n",
+    "\n",
+    "from cohere_aws import Client\n",
+    "import boto3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cohere_package = \"cohere-rerank-multilingual-v2--8b26a507962f3adb98ea9ac44cb70be1\"\n",
```

Comment:
don't forget to update

Reply:
i will have to update it tomorrow. This is the last step for the model to be deployed, let's leave as is and i'll make the change tomorrow.